package.json

```diff
@@ -98,7 +98,8 @@
     "@types/js-yaml": "^4.0.0",
     "@types/mkdirp": "^1.0.1",
     "@types/mocha": "^8.2.1",
-    "@types/node": "^14.14.31",
+    "@types/mongodb": "^4.0.7",
```

Comment:
Doesn't mongodb@4 have built-in typings?

Reply:
Maybe, I can try to remove the @types